app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -98,13 +96,18 @@ elif [ -f /etc/debian_version ]; then
     curl --silent --show-error --output /etc/apt/trusted.gpg.d/client<%= index %>.asc <%= shell_escape repo_gpg_key_url %>
   fi
 <% end -%>
-  apt-get update
-
 else
   echo "Unsupported operating system, can't add repository."
   cleanup_and_exit 1
 fi
```

Comment:
```suggestion
fi

```
While you are at it, you could add a newline such that the code blocks that configure a repository will be separated by a newline in the resulting shell script.